src/lxc/cgroups/cgroup.h

```diff
@@ -54,7 +54,11 @@ typedef enum {
  *   init's cgroup (if root).
  *
  * @container_full_path
- * - The full path to the containers cgroup.
+ * - The full path to the container's cgroup.
+ *
+ * @container_limit_path
+ * - The full path to the container's limiting cgroup. May simply point to
+ *   container_full_path.
```

Comment:
The case where they both point to the same memory is handled safely, i.e. we can't end-up double-freeing?

Reply:
I went through all the cases I found, but if you feel safer with a simple `strdup()` that's fine with me.

Reply:
Nah, it's fine.